access/Dockerfile

```diff
@@ -7,6 +7,9 @@ ARG ACCESS_PLUGIN
 
 WORKDIR /workspace
 
+# Copy .git directory needed for go build
```

Comment:
Am I right in thinking that this image will be rebuilt on every push in Drone? Otherwise, I'm not sure I understand adding the `.git` dir

Reply:
I believe it's needed for the `git describe ...` commands in the `Makefile`s

Reply:
@kbence is correct. The binaries are built with their Gitref variable set to the output of the `git describe` command. Also the goal is for these images to be built uniquely on every push to the master branch, however this PR just fixes some of the processes with ensuring the versioning is correctly reported (matching the tsh, tctl commands) across the different plugins.